doc/source/api.rst

```diff
@@ -1117,6 +1229,8 @@ Attributes and underlying data
    Panel.ftypes
    Panel.get_dtype_counts
    Panel.get_ftype_counts
```

Comment:
maybe not in this PR but we should remove Panel from api.rst

Reply:
Yeah, will do that when panel is removed.